src/index.html

```diff
@@ -3,9 +3,10 @@
 
 <head>
   <meta charset="utf-8">
-  <title>PracLearn</title>
-  <base href="/">
-
+  <!--Issue #3 Remove <title> <title>PracLearn</title> -->
```

Comment:
Please just delete this line. Commenting code out is a bad practice, it creates "zombie code".

Reply:
OK, sure.